src/web/src/components/Icon/commons/LogOut01.tsx

```diff
@@ -0,0 +1,23 @@
+import React from "react";
+
+const LogOut01 = () => {
```

Comment:
Nunca é pra pegar os icones direto da página, sempre tem q ir em `Go to main component` e pegar da página principal lá

https://www.figma.com/design/f35fmFOuUOX0JYh0J68l5j/%E2%9D%96-Untitled-UI-%E2%80%93-PRO-VARIABLES-(v5.0?node-id=3463-405222&m=dev